frontend/src/App.vue

```diff
@@ -26,21 +27,19 @@ import Loading from "vue-loading-overlay";
 import Collections from "./components/Collections.vue";
 import { RpcBrowser } from "@sap-devx/webview-rpc/out.browser/rpc-browser";
 import { RpcBrowserWebSockets } from "@sap-devx/webview-rpc/out.browser/rpc-browser-ws";
-
 function initialState() {
   return {
     collections: [],
     rpc: Object,
     messages: {},
-    showBusyIndicator: false,
+    showBusyIndicator: false
```

Comment:
why removing trailing comma here, but add it in line 42?
(This project seems to need a linter/format tool)